sandbox/src/mocks/stu3/bookOrDeferAppointment/responses/MinimalDeferral.json

```diff
@@ -22,6 +22,7 @@
   ],
   "description": "e-Referral Appointment",
   "created": "2021-06-30T13:58:47.940+00:00",
+  "comment": "There are no slots",
```

Comment:
I'm not sure this should have been added - a comment isn't present in the request, so wouldn't be returned in the response.

Reply:
I added it to the request so that this field can be displayed in the response.

Reply:
If comment isn't a mandatory field, should it be present in the "Minimum deferral" example? It's already present in the other two deferral examples.